packages/bpk-component-popover/src/__snapshots__/BpkPopover-test.js.snap

```diff
@@ -148,12 +148,12 @@ exports[`BpkPopover should render correctly with "labelAsTitle" attribute 1`] =
        
       <button
         aria-label="Close"
-        class="bpk-close-button bpk-popover__close-button"
+        class="bpk-close-button__default bpk-popover__close-button"
         title="Close"
         type="button"
       >
         <span
-          class="bpk-close-button__icon"
+          class="bpk-close-button-icon"
```

Comment:
`bpk-close-button__icon` vs `bpk-close-button-icon` should we keep the `__`?

Reply:
Well I feel that the icon isn't an extension of bpk-close-button so I wanted to make it clear that it was different to the root class name.  Where as the __default and __onDark are extensions of it.

Reply:
works for me!